src/index.html

```diff
@@ -7,8 +7,34 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="#"
+        >
+      </a>
+      <nav>
+        <ul class="nav">
+          <a class="nav_link is-active" href="#">Apple</a>
+          <a class="nav_link" href="#">Samsung</a>
+          <a class="nav_link" href="#">Smartphones</a>
+          <a class="nav_link"
+             href="#"
+             data-qa="hover"
+          >Laptops & Computers</a>
```

Comment:
If we have 3 or more attributes, then all of them must be on a new line

Reply:
<img width="283" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/129754902/07483099-f0ec-43e3-87cb-73fc76dd6d85">

Reply:
it is right?

Reply:
it is right?